constants.js

```diff
@@ -0,0 +1,9 @@
+const constants = {
```

Comment:
Use `Object.freeze` to prevent changing of the object from other places.
Also, that's a bad name. This object must describe its variables